src/library/scala/collection/convert/StreamExtensions.scala

```diff
@@ -84,8 +84,8 @@ trait StreamExtensions {
 
   implicit class MapHasParKeyValueStream[K, V, CC[X, Y] <: collection.MapOps[X, Y, CC, _]](cc: CC[K, V]) {
     private type MapOpsWithEfficientKeyStepper[K, V] = collection.MapOps[K, V, CC, _] { def keyStepper[S <: Stepper[_]](implicit shape : StepperShape[K, S]) : S with EfficientSplit }
```

Comment:
Is the shadowing of K and V here intended?

Reply:
Maybe not intended, but it's not a bug :-)

Reply:
Fixed in https://github.com/scala/scala/pull/8095/commits